### ISSUE TITLE
Can we use snp only to estimate divergence time using BPP?

### ISSUE THREAD

**Issue by xiekunwhy** [bpp/bpp#143]
Hi,

Can we use snp only (without invariant sites) to estimate divergence time using BPP? Why or why not?

Best,
Kun

**Comment by zhengluo-zz** [comment]
@xiekunwhy hello, did you try this method?

**Comment by brannala** [comment]
No, because the invariant sites are informative about the branch lengths on gene trees and thus divergence time. Using only variable sites (SNPs) will bias divergence time estimates upwards. For high coverage data the appropriate thing to do is to use bcftools to generate sequences that include invariant sites which are interpolated from the reference sequence used to assemble the reads. With lower coverage data you need to include missing data in regions where base calls are poor.
